client/src/hooks/system-status/useSystemQueue.js

```diff
@@ -0,0 +1,17 @@
+/*
+  Used in portal/client/src/workspace/SystemQueueTable.tsx 
+  for getting system queue data from core portal
+*/
+export const useSystemQueue = async (hostname) => {
+  const response = await fetch(`/api/proxy/system-monitor/${hostname}/`);
```

Comment:
This should also be data fetched from the tap portal. e.g. https://tap.tacc.utexas.edu/status/Stampede3/

Reply:
Just wanted to echo Sal's comment here

Reply:
Looks like this still needs to be addressed